activation.cpp

```diff
@@ -106,10 +143,31 @@ auto Activation::activation(Activations value) -> Activations
         using UpdateAccessKey = phosphor::software::image::UpdateAccessKey;
         UpdateAccessKey updateAccessKey(manifestPath);
 
+        using VersionClass = phosphor::software::manager::Version;
+
+        std::string versionID =
+            VersionClass::getValue(manifestPath.string(), "version");
+        std::string currVersion = versionID.substr(2, 4);
```

Comment:
Let's put a comment that the release version is expected to be in the format of XXYYYY where YYYY is the release version.